context.go

```diff
@@ -1227,6 +1227,7 @@ func (c *Context) findMatchingVariant(module *moduleInfo, possible []*moduleInfo
 }
 
 func (c *Context) addDependency(module *moduleInfo, tag DependencyTag, depName string) []error {
+
```

Comment:
I usually only insert an empty line at the top of the function if the function signature wraps to multiple lines, since gofmt will indent the wrapped signature the same as the first statement.